### PR TITLE
Doc describing first phase of repo migration

### PR DESCRIPTION
**Pull request by sarath-p**
Describes high level steps to move `spk` repo from Catalystcode to Microsoft org

### CODE REVIEW

File: technical-docs/designs/spkmove.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# Moving `SPK` repo from `Catalystcode` to `Microsoft` org
+
+| Revision | Date         | Author (s)    | Remarks       |
+| -------: | ------------ | ------------- | ------------- |
+|      0.1 | Mar-30, 2020 | Sarath, Andre | Initial Draft |
+
+## Overview
+
+This dcoument describes high level steps to move Bedrock-CLI repo (formerly
+known as `SPK`) to the MIcrosoft organization in GitHub.
```
Comment (yradsmikham) [inline technical-docs/designs/spkmove.md]: "Microsoft"

**Review by yradsmikham — APPROVED** [review]

File: technical-docs/designs/spkmove.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+1. The docs in `spk` and `bedrock` repos have references which needs to
+   modified.
+
+2) The build pipelines in Azure DevOps are referencing `yaml` files in `spk`
+   repo which needs to be modified.
+
+## Risks & Mitigations
+
+1. Docs and builds as kisted in the above section may break as part of this
```
Comment (yradsmikham) [inline technical-docs/designs/spkmove.md]: "listed"?

File: technical-docs/designs/spkmove.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
+# Moving `SPK` repo from `Catalystcode` to `Microsoft` org
+
+| Revision | Date         | Author (s)    | Remarks       |
+| -------: | ------------ | ------------- | ------------- |
+|      0.1 | Mar-30, 2020 | Sarath, Andre | Initial Draft |
+
+## Overview
+
+This dcoument describes high level steps to move Bedrock-CLI repo (formerly
```
Comment (andrebriggs) [inline technical-docs/designs/spkmove.md]: document

File: technical-docs/designs/spkmove.md (hunk 2 above)
Comment (sarath-p) [reply technical-docs/designs/spkmove.md]: Fixed.

File: technical-docs/designs/spkmove.md (hunk 1 above)
Comment (sarath-p) [reply technical-docs/designs/spkmove.md]: Fixed

File: technical-docs/designs/spkmove.md (hunk 3 above)
Comment (sarath-p) [reply technical-docs/designs/spkmove.md]: Fixed.

**Review by dennisseah — APPROVED** [review]

**Review by samiyaakhtar — APPROVED** [review]

**Pull request merged**